CMakeLists.txt

```diff
@@ -31,6 +31,7 @@ include(GNUInstallDirs)
 # Create the main library
 add_library(
   openarm_can
+  SHARED
```

Comment:
Users can change it by `-DBUILD_SHARED_LIBS=ON`: https://cmake.org/cmake/help/latest/variable/BUILD_SHARED_LIBS.html

Reply:
I see. We decided not to hardcode `SHARED` in CMakeLists.txt.
Instead, we’ll keep `add_library(openarm_can …)` and let users explicitly specify  -DBUILD_SHARED_LIBS=ON` if they need a shared library. Does my understanding sound correct about it?

Reply:
Correct.